data/meson.build

```diff
@@ -0,0 +1,36 @@
+custom_target(
+    'appdata',
+    input: 'xviewer.appdata.xml.in',
+    output: 'xviewer.appdata.xml',
+    command: [intltool_merge, '-x', '-u', po_dir, '@INPUT@', '@OUTPUT@'],
```

Comment:
Can't https://mesonbuild.com/i18n-module.html#i18nmerge_file be used for this?

Reply:
Updated, thanks!